### PR TITLE
HomeTVApp: added drawer on the left side of screen

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] N/A
[Problem] add action on swipe from left
[Solution]
 Drawer widget is standard widget to install on swipe gesture
 from the left. Several styles of Drawer in mobile profile
 needed update.

[Test]
- install app
- swipe from left side of screen

[Demo]
obraz

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
@TomaszLukawskiSam would it be possible to ad "menu" icon in appbar in upper left corner so the user knows that there is a hidden menu?
image

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam would it be possible to ad "menu" icon in appbar in upper left corner so the user knows that there is a hidden menu?
> image

Ok. I'll do it.

File: src/js/profile/mobile/widget/Drawer.js [hunk 1]
```diff
@@ -62,6 +62,8 @@
 				 */
 
 				self.options.dragEdge = 0.05;
+				self.options.width = 0.90 * window.screen.width;
```
Comment (lslachciak) [inline src/js/profile/mobile/widget/Drawer.js]: according to my checks in other aps menu takes about 75% of screen width

**Review by lslachciak — COMMENTED** [review]
Please check my comment in code and please refer in commit message to issue 
https://github.com/Samsung/TAU/issues/1253

**Comment by TomaszLukawskiSam** [comment]
> > @TomaszLukawskiSam would it be possible to ad "menu" icon in appbar in upper left corner so the user knows that there is a hidden menu?
> > image
> 
> Ok. I'll do it.

obraz

File: src/js/profile/mobile/widget/Drawer.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/Drawer.js]: Done.

**Comment by TomaszLukawskiSam** [comment]
Source of Hamburger icon: https://commons.wikimedia.org/wiki/File:Hamburger_icon.svg

**Review by lslachciak — APPROVED** [review]

**Pull request merged**